index.html

```diff
@@ -197,38 +197,39 @@ <h1 class="name">Start Bootstrap</h1>
                     <!-- The form should work on most web servers, but if the form is not working you may need to configure your web server differently. -->
                     <form name="sentMessage" id="contactForm" novalidate>
                         <div class="row control-group">
+
                             <div class="form-group col-xs-12 floating-label-form-group controls">
-                                <label for="name">Name</label>
-                                <input type="text" class="form-control" placeholder="Name" id="name" required data-validation-required-message="Please enter your name.">
+                                <label for="name">Imię</label>
+                                <input type="text" class="form-control" placeholder="Imię" id="name" required data-validation-required-message="Wpisz swoje imię.">
                                 <p class="help-block text-danger"></p>
                             </div>
                         </div>
                         <div class="row control-group">
                             <div class="form-group col-xs-12 floating-label-form-group controls">
-                                <label for="email">Email Address</label>
-                                <input type="email" class="form-control" placeholder="Email Address" id="email" required data-validation-required-message="Please enter your email address.">
+                                <label for="email">Adres email</label>
```

Comment:
Email or e-mail? Please be consitent in naming.

Reply:
I think I'm consitent. I didn't use 'email' anywhere.

Reply:
Hmm, my bad then, sorry. ;-)